### ISSUE TITLE
data preprocess question

### ISSUE THREAD

**Issue by unparalleled-ysj** [TensorSpeech/TensorFlowTTS#273]
Hi, I have a question, why revert "Correct preprocessing step, add eos_f0, eso_energy, eos_mel."

**Comment by dathudeptrai** [comment]
:))) i think the old is better. I saw the tacotron2 model stop_token overfit (zero loss), because the last mel is zero so the model ez to recognize this pattern in teacher forcing mode.
